<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cripto</class>
 <widget class="QMainWindow" name="cripto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/newPrefix/icons/dollar.png</normaloff>:/newPrefix/icons/dollar.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: century gothic;
}

QWidget{
background-color: rgb(0, 170, 127);
}

QLabel {
color:red;
background:white;
border: 1px solid red;
border-radius: 10px;
font: 57 10pt &quot;Asimov&quot;;
}

QLineEdit {
color:black;
background:white;
border: 1px solid red;
border-radius: 10px;
font: 57 15pt &quot;Asimov&quot;;
}
QComboBox{
color:black;
background:white;
border: 1px solid red;
border-radius: 10px;
font: 57 15pt &quot;Asimov&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Asimov</family>
      <pointsize>10</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Conversor De Criptomonedas</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>212</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_1">
         <property name="font">
          <font>
           <family>Asimov</family>
           <pointsize>15</pointsize>
           <weight>7</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_1"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="font">
          <font>
           <family>Asimov</family>
           <pointsize>15</pointsize>
           <weight>7</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Version No Disponible</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDescargar_Datos"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuacerca_de">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuCriptomoneda">
    <property name="title">
     <string>Conversor divisas</string>
    </property>
    <addaction name="actionIr_a_divisas"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCriptomoneda"/>
   <addaction name="menuacerca_de"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDescargar_Datos"/>
   <addaction name="separator"/>
   <addaction name="actionIr_a_divisas"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionDescargar_Datos">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/newPrefix/icons/loading.png</normaloff>:/newPrefix/icons/loading.png</iconset>
   </property>
   <property name="text">
    <string>Descargar Datos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/newPrefix/icons/cancel.png</normaloff>:/newPrefix/icons/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/newPrefix/icons/information.png</normaloff>:/newPrefix/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionIr_a_divisas">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/newPrefix/icons/dollar.png</normaloff>:/newPrefix/icons/dollar.png</iconset>
   </property>
   <property name="text">
    <string>Ir a conversor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>cripto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
